src/components/mols/CreditCardPayment/index.tsx

```diff
@@ -69,17 +70,30 @@ const CreditCardPayment: React.FC<ICreditCardPaymentProps> = ({
       ),
   });
   const initialValues: ICreditCardPayment = {
-    cardNumber: "",
+    creditCard: "",
     cvv: "",
-    nameOnCard: "",
+    fullName: "",
     cardMonth: "1",
     cardYear: String(new Date().getFullYear()),
     TermsOfUse: true,
   };
 
   const handleSubmit = async (values: ICreditCardPayment) => {
+    const fragmentedName = values.fullName.split(" ");
+    const firstName = fragmentedName[0];
+    const lastName = fragmentedName.slice(1).join(" ");
+
+    const iuguData = {
+      number: values.creditCard,
+      first_name: firstName,
+      last_name: lastName,
+      verification_value: values.cvv,
+      month: values.cardMonth,
+      year: values.cardYear,
+    };
     setLoading(true);
-    console.log(values);
+    Iugu.setTestMode(true);
```

Comment:
Change this to .ENV and set true or false in .env file =)

Reply:
Done